src/algorithms/contractions/ctmrg_contractions.jl

```diff
@@ -1,4 +1,4 @@
-const CTMRGEdgeTensor{S} = AbstractTensorMap{S,3,1}
+const CTMRGEdgeTensor{S,N} = AbstractTensorMap{S,N,1}
```

Comment:
```suggestion
const CTMRGEdgeTensor{S,N} = AbstractTensorMap{S,N,1}
const CTMRG_PEPS_EdgeTensor{S} = CTMRGEdgeTensor{S,3}
const CTMRG_PF_EdgeTensor{S} = CTMRGEdgeTensor{S,2}
```
I think it could be nice to have these (unexported) type aliases in the function signatures.
The naming is annoying, as just gluing everything together is ridiculously ugly. I would normally not capitalize acronyms, and use capitalization conventions of CamelCasing etc based on the type of variable name, but it might be too late for that now. `CTMRG_EdgeTensor` and `CTMRG_CornerTensor` might also be reasonable?

Reply:
I think your suggestion is fine for internal use. I don't think switching to `CTMRG_EdgeTensor` is worth it now. I actually thinking we could probably just drop the `CTMRG` and just use `EdgeTensor` and `CornerTensor`, since this does not sound that much less clear to me and these could at some point also be used in other contraction algorithms. But we can discuss this some other time.